lug/config.yaml

```diff
@@ -14,10 +14,162 @@ dummy:
+    <<: *oneshot_common
+  #deepin-cd
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: https://deepin.c3sl.ufpr.br/deepin-cd/
+    interval: 21600
+    path: /srv/deepin-cd
+    name: deepin-cd
+    <<: *oneshot_common
+  #docker-ce
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: https://download.docker.com/
+    interval: 86400
+    path: /srv/docker-ce
+    name: docker-ce
+    <<: *oneshot_common
+  #fedora
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: rsync://download-ib01.fedoraproject.org/fedora-enchilada/linux/
+    interval: 21600
+    path: /srv/fedora
+    name: fedora
+  #fedora-secondary
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: rsync://download-ib01.fedoraproject.org/fedora-secondary/
+    interval: 21600
+    path: /srv/fedora-secondary
+    name: fedora-secondary
+    <<: *oneshot_common
+  #linuxmint
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: rsync://mirrors.kernel.org/linuxmint-packages/
+    interval: 21600
+    path: /srv/linuxmint
+    name: linuxmint
+    <<: *oneshot_common
+  #linuxmint-cd
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: rsync://mirrors.kernel.org/linuxmint/
+    interval: 21600
+    path: /srv/linuxmint-cd
+    name: linuxmint-cd
+    <<: *oneshot_common
+  #mageia
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: rsync://mirrors.kernel.org/mageia/
+    interval: 21600
+    path: /srv/mageia
+    name: mageia
+    <<: *oneshot_common
+  #opensuse
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: rsync://fr2.rpmfind.net/linux/opensuse/
+    interval: 43200                                       #may be wrong
+    path: /srv/opensuse
+    name: opensuse
+    <<: *oneshot_common
+  #openvz
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: rsync://download.openvz.org/openvz-download
+    interval: 43200                                       #may be wrong
+    path: /srv/openvz
+    name: openvz
+    <<: *oneshot_common
+  #remi
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: rsync://rpms.remirepo.net
+    interval: 43200
+    path: /srv/remi
+    name: remi
+    <<: *oneshot_common
+  #scientific
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: rsync://mirror.aarnet.edu.au/pub/scientific/
+    interval: 21600                                       #may be wrong
+    path: /srv/scientific
+    name: scientific
+    <<: *oneshot_common
+  #ubuntu
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: rsync://archive.ubuntu.com/ubuntu/
+    interval: 21600
     path: /srv/ubuntu
     name: ubuntu
     <<: *oneshot_common
+  #ubuntu-cdimage
+  - type: shell_script 
+    script: /worker-script/rsync.sh
+    source: rsync://cdimage.ubuntu.com/cdimage/
+    interval: 21600
+    path: /srv/ubuntu-cdimage
```

Comment:
Please make this consistent with current ftp.sjtu configuration. Use `ubuntu-cd` instead.

Reply:
haven't find the upstream of "ubuntu-cd"

Reply:
Just change the name to `ubuntu-cd` to keep our site compatible with ftp.sjtu

Reply:
Also the path

Reply:
Ok, fixed